modules/Books.js

```diff
@@ -0,0 +1,70 @@
+import Collections from './Collections.js';
+
+import { DateTime } from './luxon.min.js';
+
+const bookContainer = document.querySelector('.book-container');
+const date = document.querySelector('.date');
+
+const collections = new Collections();
+
+export default class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.timeStamp = Book.timeStamp();
+  }
+
+  addBook() {
+    collections.addBook(this);
+    this.render();
+  }
```

Comment:
- [ ] Please it would be better if make these methods as `Arrow functions` instead of `normal functions` that will make use the Es6 features